tt-train/CMakeLists.txt

```
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.18)
```

README says v3.30, let's hint to CMake the same.

```suggestion
cmake_minimum_required(VERSION 3.18..3.30)
```